airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -130,6 +131,14 @@ public void setFeedback(final UUID workflowId) throws JsonValidationException, C
     persistence.writeConfig(ConfigSchema.STANDARD_WORKSPACE, workspace.getWorkspaceId().toString(), workspace);
   }
 
+  public ActorDefinition getActorDefinition(final UUID actorDefId, final ActorType actorType)
+      throws JsonValidationException, IOException, ConfigNotFoundException {
+    return persistence.getConfig(
+        actorType == ActorType.SOURCE ? ConfigSchema.STANDARD_SOURCE_DEFINITION : ConfigSchema.STANDARD_DESTINATION_DEFINITION,
```

Comment:
Nit:  this could be simplified by adding an abstract method to the enum and forcing each enum value to implement.  Something like:

```
public enum ActorType {
    SOURCE {
        public ConfigSchema getSchema() { return ConfigSchema.STANDARD_SOURCE_DEFINITION; }
    },
    ....

     public abstract ConfigSchema getSchema();
}
```

It also doesn't have to be an abstract method -- it could just be a method that some of the enum values override with the standard one returning the standard schema destination definition.   Definitely not something that should hold this change up, but something to consider to reduce these types of tertiary statements.

Reply:
oh. that's a nice thought. i'll play with that when i get back to this PR!